uui-components/src/pickers/PickerInputBase.tsx

```diff
@@ -64,6 +64,10 @@ export type PickerInputBaseProps<TItem, TId> = PickerBaseProps<TItem, TId> & ICa
     fixedBodyPosition?: boolean;
 
     portalTarget?: HTMLElement;
+
+    /** Styles for input and body components */
```

Comment:
add the comment for each prop and let's make it more specific, look to the description of general 'cx' props and use the same structure and wordings.

Reply:
done